io.openems.edge.bridge.modbus/src/io/openems/edge/bridge/modbus/ConfigTcp.java

```diff
@@ -33,5 +33,8 @@
 	@AttributeDefinition(name = "Invalidate elements after how many read Errors?", description = "Increase this value if modbus read errors happen frequently.")
 	int invalidateElementsAfterReadErrors() default 1;
 
+	@AttributeDefinition(name = "Interval between accesses (in milliseconds)", description = "Cycle skipping will be calculated from this value and core cycle time.")
+	int intervalBetweenAccesses() default 0;
```

Comment:
I would rename this to `minIntervalBetweenAccesses()`.

Description for unit can be simply "[ms]" instead of "(in milliseconds)" - see this example: https://github.com/OpenEMS/openems/blob/develop/io.openems.edge.core/src/io/openems/edge/core/cycle/Config.java#L13

Reply:
Thanks! I will fix them.